cmd/conduit/main.go

```diff
@@ -71,7 +71,16 @@ func parseConfig() conduit.Config {
 
 		connectorsDir = flags.String("connectors.path", "./connectors", "path to standalone connectors directory")
 
-		pipelinesDir = flags.String("pipelines.path", "./pipelines", "path to the directory that has the yaml pipeline configuration files, or a single pipeline configuration file")
+		pipelinesDir = flags.String(
+			"pipelines.path",
+			"./pipelines",
+			"path to the directory that has the yaml pipeline configuration files, or a single pipeline configuration file",
+		)
+		pipelinesExitOnError = flags.Bool(
+			"pipelines.exit-on-error",
+			false,
+			"Exit Conduit if any pipeline provisioned through a file experiences an error while running.",
```

Comment:
The description doesn't fit now, we should mention that Conduit also exits if any pipeline stops because of an error.

Also a nitpick: the description should start with a lowercase "e" and not end with "." to keep it consistent with other flags.